.github/CONTRIBUTING.md

```diff
@@ -35,8 +39,9 @@ Follow the [Meme Battle styleguide](https://github.com/Meme-Battle/styleguide/bl
       │   │           └──styles.module.scss   
       │   ├── utils/ # reusable hooks
       │   │   └── # Start Hook name with capital letter 
```

Comment:
there are no examples on this folder, and we created one completely different than the expected
`GetLocales.js`

I would change to `getLocales.ts` (don't forget typescript)